raiden/transfer/node.py

```diff
@@ -471,6 +472,21 @@ def handle_token_network_action(
 
         events = iteration.events
 
+        if type(state_change) == ContractReceiveChannelClosed:
```

Comment:
please, move this to its own handle.

Reply:
Ok, did it like that because all the token network events are processed the same way, so I put it in the same handler. Will split this in a separate handler.